lxd/migrate_instance.go

```diff
@@ -41,10 +39,7 @@ func newMigrationSource(inst instance.Instance, stateful bool, instanceOnly bool
 	secretNames := []string{api.SecretNameControl, api.SecretNameFilesystem}
 	if stateful && inst.IsRunning() {
 		if inst.Type() == instancetype.Container {
-			_, err := exec.LookPath("criu")
-			if err != nil {
-				return nil, migration.ErrNoLiveMigrationSource
-			}
+			return nil, errors.New("Live migration is not supported for containers")
```

Comment:
Should we be using api.StatusErrorf with an appropriate HTTP response for these "not supported" errors (and elsewhere in this PR)?